lib/error/ble_error.dart

```diff
@@ -0,0 +1,36 @@
+abstract class _Metadata {
```

Comment:
Change the name to `_BleErrorMetadata`. We use only one namespace currently, and using `_Metadata` will create conflics sooner than later.